libs/sdk-backend-tiger/src/backend/workspace/execution/executionResult.ts

```diff
@@ -126,6 +161,31 @@ export class TigerExecutionResult implements IExecutionResult {
             return new TigerDataView(this, result, this.dateFormatter);
         });
     };
+
+    private async handleExportResultPolling(
+        client: ITigerClient,
+        payload: ActionsApiGetTabularExportRequest,
+    ): Promise<IExportResult> {
+        for (let i = 0; i < MAX_POLL_ATTEMPTS; i++) {
+            const result = await client.export.getTabularExport(payload, {
+                transformResponse: (x) => x,
+            });
+
+            if (result?.status === 200) {
+                return {
+                    uri: result?.config?.url || "",
+                };
+            }
+
+            await new Promise((resolve) => setTimeout(resolve, DEFAULT_POLL_DELAY));
+        }
+
+        return Promise.reject(
```

Comment:
Nitpick: instead of `return Promise.reject(foo)` you can just `throw foo` since you are in an async function.